src/screens/safe-overview/components/danger-zone/danger-zone.component.tsx

```diff
@@ -0,0 +1,32 @@
+import { Box, Text } from 'components/primitive';
+import { StyledRow, DangerZoneButton } from './danger-zone.component.styles';
+
+export const DangerZone = () => {
+  return (
+    <Box color='white' padding={4} width={43.3}>
+      <Text variant='contentHeader' tx='safeOverViewPage.dangerZone' />
+      <Box marginTop={2.4}>
+        <StyledRow>
```

Comment:
Why are not using `Box` with row property ?

Reply:
changed in the latest commit.